tutorials/monitoring/setup/index.md

```diff
@@ -41,27 +41,25 @@ scinstance --> pulse
 
 ```
 
-![the above renders like this](component-overview.png)
+An NServiceBus system monitored using the Particular Software Platform will push data to three main queues:
 
```

Comment:
@mikeminutillo Sorry, I don't understand this sentence. NSB is part of the Particular platform. Should we be more specific and mentioned to tools?

Reply:
@YifatS or drop the reference to the PSP?

> An NServiceBus system being monitored will push data to three main queues:

Reply:
Or

> A running NServiceBus system can be configured to send data about it's operation to three main queues:

Reply:
I vote for the latter :)